zmq.hpp

```diff
@@ -1107,7 +1108,7 @@ namespace zmq
     private:
         void *poller_ptr;
         std::vector<zmq_poller_event_t> poller_events;
-        std::map<socket_t*, handler_t> handlers;
+        std::unordered_map<socket_t*, handler_t> handlers;
```

Comment:
I don't see how this change is related to the Windows build support (maybe #include <map> was missing?).